test/6_branch.js

```diff
@@ -835,17 +835,6 @@ describe('Branch', function() {
 		});
 	});
 
-	describe('sendSMS', function() {
```

Comment:
There's a reference to SMS in https://github.com/BranchMetrics/web-branch-deep-linking-attribution/blob/master/docs/1_intro.md?plain=1#L23

Reply:
Fixed.Thanks.